functions/package.json

```diff
@@ -30,7 +32,7 @@
     "eslint": "^8.9.0",
     "eslint-config-google": "^0.14.0",
     "eslint-plugin-import": "^2.25.4",
-    "firebase-functions-test": "^3.0.0",
+    "firebase-tools": "11.2.2",
```

Comment:
removing `firebase-functions-test` because we aren't using it and pinning `firebase-tools` due to an issue with concurrent emulator network calls: https://github.com/firebase/firebase-tools/issues/5134